src/Extension.ml

```diff
@@ -87,6 +94,15 @@ module Instance = struct
          restart the server.";
     Ok ()
 
+  let openTerminal ?toolchain t () =
```

Comment:
`()` isn't necessary because we have a positional argument `t` here.

Reply:
Oh, it's because it's registered as a command later. Never mind

Reply:
The `()` is needed for it to be a `unit -> unit` function for when it is used as a handler for `Vscode.Commands.register`.

I can remove the `()`, but that will require replacing its `registerCommands` entry with a lambda.